src/WebUI/Pods/PodOverview.tsx

```diff
@@ -175,7 +175,15 @@ export class PodOverview extends BaseComponent<IPodOverviewProps> {
 
     private static _renderImageCell = (rowIndex: number, columnIndex: number, tableColumn: ITableColumn<any>, tableItem: any): JSX.Element => {
         const { key, value, valueTooltipText, imageId, showImageDetails } = tableItem;
-        const hasImageDetails: boolean = StoreManager.GetStore<ImageDetailsStore>(ImageDetailsStore).hasImageDetails(imageId);
+        const imageDetailsStore = StoreManager.GetStore<ImageDetailsStore>(ImageDetailsStore)
```

Comment:
rendering should be really use the local data and not depend upon complex items like store etc, otherwise this will become really costly. even in the tableItem we can send some property to indicate whether image processing is done or not.